src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/metrikker/besøksstatistikk/SammenligningEvent.java

```diff
@@ -3,14 +3,9 @@
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
-import no.nav.arbeidsgiver.sykefravarsstatistikk.api.felles.Fnr;
+import no.nav.arbeidsgiver.sykefravarsstatistikk.api.felles.*;
```

Comment:
Jeg er ikke en stor tilhenger av dette men noen mener at `widcard imports` bør unngås og kan lede til feil som er vanskelig å debugge (ref https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad og denne http://www.javadude.com/posts/20040522-import-on-demand-is-evil/ )

Reply:
Da i praksis prøver jeg å unngå `wildcard`

Reply:
God praksis synes jeg. Denne importeringen ble endret av IntelliJ, er det noen måte å skru av wildcard-imports på?